index.html

```diff
@@ -38,8 +41,190 @@ <h1>I'm Mucha <br> Glad to see <br> you!</h1>
                 <li><a href="#"><img src="images/medium.svg" alt="medium"> </a></li>
             </ul>
         </section>
+
+
+    </div>
+
+    <!-- card 1 -->
+    <div class="work-container">
+        <div class="portfolio-img">
+            <img src="images/Snapshoot-Portfolio-1.png" alt="">
+        </div>
+
+        <div class="work-content">
+
+
+            <h1 class="tonic">Tonic</h1>
+
+            <ul class="prop">
+                <li class="can">CANOPY</li>
+                <li class="bac"><img src="images/Counter.png" alt=""> Back End Dev</li>
+                <li class="num"><img src="images/Counter.png" alt=""> 2015</li>
+            </ul>
+
+            <p class="tonic-def">A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups
+                required.</p>
+
+            <ul class="stack">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="js">javaScript</li>
+            </ul>
+
+            <div class="button">
+                <button type="button">See Project</button>
+            </div>
+        </div>
+    </div>
+
+
+    <!-- card-2 -->
+    <div class="work-container2">
+        <div class="portfolio-img">
+            <img src="images/Snapshoot-Portfolio-2.png" alt="">
+        </div>
+
+        <div class="work-content">
+
+
+            <h1 class="tonic">Multi-Post<br>Stories</h1>
+
```

Comment:
- [ ] You have done great by ensuring that the heading uses `hx` HTML tag 👏 . However, in this section let's consider using `h2-h6`. This is because to be [Semantically](https://www.w3schools.com/html/html5_semantic_elements.asp) correct, the page should have only one `h1` which is the main heading of the page. In addition, doing this increases the accessibility of the page 👍 .